WebKitBrowser/CMakeLists.txt

```diff
@@ -54,7 +54,9 @@ set(PLUGIN_WEBKITBROWSER_TRANSPARENT "false" CACHE STRING "Set transparency")
 set(PLUGIN_WEBKITBROWSER_STARTURL "about:blank" CACHE STRING "Initial URL for WebKitBrowser plugin")
 set(PLUGIN_WEBKITBROWSER_USERAGENT "" CACHE STRING "User agent string")
 set(PLUGIN_WEBKITBROWSER_MEMORYPROFILE "512m" CACHE STRING "Memory Profile")
-set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE "databaseprocess:50m,networkprocess:100m,webprocess:300m,rpcprocess:50m" CACHE STRING "Memory Pressure")
+set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE "" CACHE STRING "Memory Pressure")
+set(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE_WEBPROCESSLIMIT "" CACHE STRING "Memory Pressure Webprocess Limit")
```

Comment:
Just to make sure, I discussed with Ozgur, better not to have a default in the new situation but use webkits one. So let's keep it like this

Reply:
done.